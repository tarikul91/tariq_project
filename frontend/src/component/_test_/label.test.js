import { render, screen,cleanup } from "@testing-library/react";
import Label from "component/label";
import renderer from "react-test-renderer"
afterEach(cleanup)

test('Label test',()=>{
    render(<Label className='form-label'  htmlFor="userName">Label name</Label>)
    expect(screen.getByTestId('label')).toBeInTheDocument()
    expect(screen.getByTestId('label')).toHaveClass('form-label')
    expect(screen.getByTestId('label')).toBeInTheDocument()
})
it('label snapshot',()=>{
    const tree = renderer.create(<Label className='form-label'>Label name</Label>).toJSON()
    expect(tree).toMatchSnapshot()
})