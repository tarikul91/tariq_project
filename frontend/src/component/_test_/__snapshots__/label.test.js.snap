// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`label snapshot 1`] = `
<label
  className="form-label"
  data-testid="label"
>
  Label name
</label>
`;
